plugins/cairo-lang-macro-stable/src/lib.rs

```diff
@@ -0,0 +1,51 @@
+use std::ffi::CString;
+use std::os::raw::c_char;
+
+/// Token stream.
+///
+/// This struct implements FFI-safe stable ABI.
+#[repr(C)]
+#[derive(Debug)]
+pub struct StableTokenStream(pub *mut c_char);
```

Comment:
```suggestion
pub struct StableTokenStream(*mut c_char);
```

wasn't the purpose of stable ABI to hide this fact and force users to only use public methods?